editor/src/components/canvas/ui-jsx-canvas.spec.tsx

```diff
@@ -241,6 +241,96 @@ export { ToBeDefaultExported as default }`,
     )
   })
 
+  it('renders a canvas with a component wrapped in a function that has more that one parameter', () => {
+    testCanvasRender(
```

Comment:
I think it's much more readable to use `testCanvasRenderInline` combined with an inline snapshot, that way we can better see which snapshot corresponds to which test (which helps catch regressions when updating snapshots) and we don't need to snapshot the spy output which has become uselessly large and a burden to maintain

Reply:
I think I was going with what's already in this file, so I'll see about changing these tests.

Reply:
Made this change in 1645db31b.